### DISCUSSION TITLE
Feature request: set layer / viewport PLOT OPTIONS

### DISCUSSION THREAD

**Discussion by subsurfaceiodev** [mozman/ezdxf#773] [Ideas]
Hi there ! Sorry to bother you again. Is it currently possible to set some PLOT OPTIONS of a specific layer / viewport ? For example, setting it to "Plot transparency" by default. We were exploring a sample .dxf file and found that the "PLOTTRANSPARENCY" (0 or 1) variable is used for this.

**Comment by mozman** [comment]
I've added this feature to the todo list, but it's a low priority for me and won't be added in the near future, if ever.

**Reply by subsurfaceiodev** [reply]
Alright ! Thanks for the attention 😊
